src/main/java/HM04/Bank.java

```diff
@@ -0,0 +1,16 @@
+package HM04;
+
+
+class Bank <T extends BasicInterface> {
+
```

Comment:
Main reason use generic on class level if you have fields of this generic type. For example -private T;